### PR TITLE
Wt2 new format

### PR DESCRIPTION
**Pull request by pb82**
Walkthrough 2 in the new format

### CODE REVIEW

File: public/walkthroughs/walkthrough-2/walkthrough.adoc [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+<img src="/images/wt2.png" class="img-responsive" alt="integration">
+++++
+
+[time=5]
+
+== Deploying the Fuse Aggregation App
+
+The Fuse Aggregation App provides information about flights, you deploy it using Launcher.
+
+[type=taskResource]
```
Comment (pb82) [inline public/walkthroughs/walkthrough-2/walkthrough.adoc]: At the moment we have to repeat the task resources for every task. I've added an action item to address this: `Allow walkthrough resources to be specified on the walkthrough overview page (e.g. by providing a [type=walkthroughResource] annotation)`

**Comment by pb82** [comment]
@finp i can still add the `//include::<original doc>[ ]` annotations if you think it makes sense having them

File: public/walkthroughs/walkthrough-2/walkthrough.adoc [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+. Select the *Configure APIcast* button.
+
+. Select the *Update & test in Staging Environment* button at the bottom again.
+
+. Check that a success message is displayed, and a green line along the left side of the page.
+****
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
```
Comment (finp) [inline public/walkthroughs/walkthrough-2/walkthrough.adoc]: @pb82  think you're missing docs/modules/ROOT/pages/_partials/fuse-aggregation-app-endpoint-activedocs.adoc here

Comment (finp) [reply public/walkthroughs/walkthrough-2/walkthrough.adoc]: ie in prod, it looks like https://i.imgur.com/wPiP74w.png

Comment (pb82) [reply public/walkthroughs/walkthrough-2/walkthrough.adoc]: you're right, adding now

**Pull request merged**